### PR TITLE
(PUP-7042) Mark strings in context

### PR DESCRIPTION
**Pull request by Magisus**
This commit marks user-facing error and info strings in
`lib/puppet/context.rb` and `lib/puppet/context/*` for translation.

### CODE REVIEW

**Comment by Magisus** [comment]
Closing and reopening to fix Travis.

**Comment by Magisus** [comment]
Closing and reopening to fix CI.

File: lib/puppet/context.rb [hunk 1]
```diff
@@ -77,7 +77,7 @@ def restore(name)
     if @ignores.include?(name)
       @ignores.delete(name)
     else
-      raise UndefinedBindingError, "no '#{name}' in ignores #{@ignores.inspect} at top of #{@stack.inspect}"
+      raise UndefinedBindingError, _("no '%{name}' in ignores %{ignores} at top of %{stack}") % { name: name, value0: @ignores.inspect, value1: @stack.inspect }
```
Comment (MosesMendoza) [inline lib/puppet/context.rb]: Looks like we're referencing the wrong variable placeholders here.

File: lib/puppet/context/trusted_information.rb [hunk 2]
```diff
@@ -45,7 +45,7 @@ def self.remote(authenticated, node_name, certificate)
     if authenticated
       extensions = {}
       if certificate.nil?
-        Puppet.info('TrustedInformation expected a certificate, but none was given.')
+        Puppet.info(_('TrustedInformation expected a certificate, but none was given.'))
```
Comment (McdonaldSeanp) [inline lib/puppet/context/trusted_information.rb]: TrustedInformation seems like a keyword, should we leave a comment to leave it out of translation?

Comment (Magisus) [inline lib/puppet/context/trusted_information.rb]: I'm gonna assume camel case will be left alone, same as `_` and `::`.

Comment (McdonaldSeanp) [inline lib/puppet/context/trusted_information.rb]: Yep, that makes sense

**Pull request merged**